config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { registrations: 'registrations' }
```

Comment:
We can use the newer syntax for hashes with symbol keys i.e. `controllers: { ... `